src/_guides/bump-sh-tutorials/2024-05-21-make-your-apis-discoverable-with-apisjson.md

```diff
@@ -0,0 +1,131 @@
+
+For example, an organization with one mega-API might decide their API has become too much of a monolith, and start splitting it up. Creating a new Hub and renaming that one API to "Legacy API" will provide a space for the new APIs as they pop up.
+
+How you use them is up to you, but to work with APIs.json you'll want to make a Hub.
+
+![](/images/guides/apis-json/new-hub.png)
+
+Then add some APIs to that Hub on the API's Settings page.
+
+![](/images/guides/apis-json/add-api-to-hub.png)
+
+Now head over to the Dashboard and click View Hub, and you'll be taken to a URL like `https://bump.sh/<org-slug>/hub/<hub-slug>`. To get the APIs.json endpoint you can append `/apis.json` onto the end of that URL, to produce `https://bump.sh/<org-slug>/hub/<hub-slug>/apis.json`.
+
+![](/images/guides/apis-json/apisjson.png)
+
+Great! Now, the first thing we should do with this APIs.json is use it to help people find our APIs by submitting our APIs to APIs.io, "the search engine for APIs".
+
+Head over to [APIs.io: Add](https://apis.io/add/), and paste your link into the URL box.
+
+![](/images/guides/apis-json/submit-apisio.png)
+
+If you want to automate the update, then you can work with the [APIs.io API](https://developer.apis.io/documentation/), or [grab one of their SDKs](https://developer.apis.io/sdks/), but it may well be easier to just submit the form.
+
+## Other Uses for APIs.json
+
+APIs.json has been around for a decade, but is only starting picking up steam recently thanks to tooling starting to support it. Support is still limited, but there are countless potential use cases that can be handled with this specification.
+
+Bump.sh customers are already using it for quality assurance on their developer portals, making sure all of their internal APIs are published, and available at all times. Using Continuous Integration they can check the generated APIs.json, and make sure their API documentation is there and visible on the expect URLs.
+
+There are also clear implications for monitoring, if each API can advertise its `/health` endpoints you can pass that monitoring software, or even just ping each endpoint yourself to look for issues.
+
+Another great idea from Bruno Pedro:
+
+> Imagine if a popular IDE such as VSCode would automatically load all API information just by knowing the domain you wanted to interact with. It could then present all the known APIs for the domain along with any relevant information. Opening one of the APIs would access its machine-readable definition and could automatically generate both human-readable documentation and code. -- **Source:** ["Using APIs.json to Create a Catalog of APIs"](https://apichangelog.substack.com/p/using-apisjson-to-create-a-catalog), Bruno Pedro, The API Changelog, May 6 2024
```

Comment:
Not sure if it is the intended display, or if you wanted something different?
![image](https://github.com/bump-sh/docs/assets/7700560/241cc11e-1c9a-4969-9b63-cc606f150ea5)
```suggestion
> Imagine if a popular IDE such as VSCode would automatically load all API information just by knowing the domain you wanted to interact with. It could then present all the known APIs for the domain along with any relevant information. Opening one of the APIs would access its machine-readable definition and could automatically generate both human-readable documentation and code.

  -- **Source:** ["Using APIs.json to Create a Catalog of APIs"](https://apichangelog.substack.com/p/using-apisjson-to-create-a-catalog), Bruno Pedro, The API Changelog, May 6 2024
```

Reply:
That's not meant to be in there anymore, I killed it off in later changes. Just pushed them now.